mpxe/sims/mcp23008.py

```diff
@@ -27,7 +28,13 @@ def update_pin_state(self, proj, pin, state):
         mcp23008_mask = mcp23008_pb2.MxMcp23008Store()
         mcp23008_mask.state[pin] = 1
 
-        proj.write_store(mcp23008_msg, mcp23008_mask, stores_pb2.MxStoreType.MCP23008)
+        mcp23008_update = StoreUpdate(
```

Comment:
nit: if you take out the `mcp23008_` prefix to msg mask and update, maybe this'll all fit on one line? I don't think the prefix is necessary